src/spesh/pea.c

```diff
@@ -141,7 +141,7 @@ static MVMuint16 get_deopt_materialization_info(MVMThreadContext *tc, MVMSpeshGr
         return alloc->deopt_materialization_idx;
     }
     else {
-        MVMSpeshPEAMaterializeInfo mi;
+        MVMSpeshPEAMaterializeInfo mi = {0};
```

Comment:
All the fields are set, but I'd move this declaration down to line 160, close to it's initialization.

We can leave a comment in its place to document the `else` block here if that helps.